app/routes/__public/oauth/$id.tsx

```diff
@@ -221,8 +222,8 @@ const getAnonymousAccessToken = async ({
         uuid: sql`UUID()`,
         value: response.body.accessToken,
         code,
-        installationId: searchParams.installation_id,
-        userId: searchParams.installation_id,
+        installationId: installation_id || "",
```

Comment:
See feedback in other pr - presumably on a reauth flow we could get installation id from a previous record